src/smfc/go_generator.h

```diff
@@ -41,10 +41,6 @@ class go_generator : public generator {
     generate_header_includes();
     generate_header_services();
 
-    // for Go make sure that the directories exist
```

Comment:
so many hours

Reply:
thanks!

Reply:
@crackcomm  - I think this hunk is a regression. 

if you have an flatbuffers package of `foo.bar.baz` it should create `/foo/bar/baz/*_generated.go`

Can you comment on go experience here. I think i need to revert this  directory creation

Reply:
@crackcomm  - so i fixed the bug upstream for consistency on https://github.com/smfrpc/smf/commit/d20866d5b16ffa251bd0b0bbff4d777e8480df7c

however the previous behavior matches that of flatc - so my incline is to keep the old behavior. 

Thoughts?

Reply:
It was creating an empty directory with files outside of it...